src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier supplier = new FigureSupplier();
```

Comment:
Please ensure that 'FigureSupplier' is not an abstract class that sets behavior for other classes. It should be a concrete class that provides specific functionality.